packages/router-generator/src/config.ts

```diff
@@ -13,6 +13,7 @@ export const configSchema = z.object({
   disableTypes: z.boolean().optional().default(false),
   addExtensions: z.boolean().optional().default(false),
   disableLogging: z.boolean().optional().default(false),
+  generateManifest: z.boolean().optional().default(true),
```

Comment:
I believe this should follow the naming above and should be `disableManifestGeneration` or something to that effect.